deploy.py

```diff
@@ -13,105 +13,86 @@ def add_cli_args():
                                  "openlab-ha"],
                         help='OpenLab deployment type',
                         )
+    parser.add_argument('--action',
+                        choices=["deploy", "new-slave", "switch-role",
+                                 "show-graph", "show-ip"],
+                        default="deploy",
+                        help="The action that labkeeper supports.\n"
+                             "'deploy': create a new OpenLab CI environment.\n"
+                             "'new_slave': create new slave nodes base on "
+                             "current master nodes.\n"
+                             "'switch_role': switch the master/slave role of "
```

Comment:
and also , this help message is not enough, e.g. the new_slave required `--new-ips`